analyzers/src/SonarAnalyzer.CSharp/Rules/CollectionEmptinessChecking.cs

```diff
@@ -138,11 +89,61 @@ private static bool TryGetCountCall(ExpressionSyntax expression, SemanticModel s
             return true;
         }
 
-        private static bool IsMethodCountExtension(IMethodSymbol methodSymbol)
+        private static bool IsMethodCountExtension(IMethodSymbol methodSymbol) =>
+            methodSymbol.Name == "Count" &&
+            methodSymbol.IsExtensionMethod &&
+            methodSymbol.ReceiverType != null;
```

Comment:
To follow our coding style
```suggestion
        private static bool IsMethodCountExtension(IMethodSymbol methodSymbol) =>
            methodSymbol.Name == "Count" 
            && methodSymbol.IsExtensionMethod 
            && methodSymbol.ReceiverType != null;
```